cypress/e2e/dfeAdmin/manageKPIReport/dfeAdmin-KPI-report.spec.js

```diff
@@ -0,0 +1,128 @@
+
+        // And the week 1 searches result exists
+        cy.get('[data-testid="searches-week1"]').should('exist');
+        // And the week 2 searches result exists
+        cy.get('[data-testid="searches-week2"]').should('exist');
+        // And the week 3 searches result exists
+        cy.get('[data-testid="searches-week3"]').should('exist');
+        // And the week 4 searches result exists
+        cy.get('[data-testid="searches-week4"]').should('exist');
+    })
+
+
+    it('Clicking on the back button when on the find metrics page', () => {
+        //Clicking on the back button
+        cy.xpath('//a[normalize-space()=\'Back\']').click();
+        //Verify the title of the manage account page
+        cy.title().should('eq', 'DfE Admin - Manage family support services and accounts - GOV.UK')
+    })
+
+    it('Clicking on the connect metrics performance tab', () => {
+        //Click to view the Performance data for Connect families to support
+        cy.xpath('//a[normalize-space()=\'Connect families to support\']').click();
+        cy.title().should('eq', 'Performance data for Connect families to support - Manage family support services and accounts - GOV.UK')
+    });
+
+
+    it('correct page headings displayed for Connect report', () => {
+        const expectedPageHeading = "Performance data for Connect families to support";
+        const expectedSubtext = "Data about local authority services for Connect families to support (Connect).";
+        const expectedFilterHeader = "Services performance";
+
+        //Click to view the Performance data for Connect families to support
+        cy.xpath('//a[normalize-space()=\'Connect families to support\']').click();
+
+        // Then the page heading for the Find report is displayed correctly
+        cy.checkPageHeading("h1", expectedPageHeading);
+
+        // And the page subtext is displayed correctly
+        cy.checkPageHeading('#main-content > div.govuk-grid-row.govuk-\\!-margin-bottom-8 > div > p', expectedSubtext);
+
+        // And the filter heading is displayed correctly - the xpath could not be helped there are no ids/data test ids
+        cy.checkPageHeading('#main-content > div:nth-child(2) > div.govuk-grid-column-one-quarter > h2', expectedFilterHeader);
+    })
+
+
+    it('Verify the title and content of the overall totals for Connect table', () => {
+            const overallTotalsTitles  = "Overall totals for Connect";
+            const totalsForTheLast7DaysTitle = "Totals for the last 7 days";
+            const searchesInTheLast4WeeksTitle = "Searches in the last 4 weeks";
+
+        //Click to view the Performance data for Connect families to support
+        cy.xpath('//a[normalize-space()=\'Connect families to support\']').click();
+        //verifying the overall total for find text
+        cy.checkElementContainsTextXpath('//h2[normalize-space()=\'Overall totals for Connect\']', overallTotalsTitles);
```

Comment:
refactor: Can we use the new `data-testid`'s defined in [FHG-5439](https://dfedigital.atlassian.net/browse/FHG-5439) instead of these XPath thingies

Reply:
Done